tests/unit/test_cassettes.py

```diff
@@ -29,6 +29,19 @@ def test_cassette_load(tmpdir):
     assert len(a_cassette) == 1
 
 
+def test_cassette_load_nonexistent():
+    a_cassette = Cassette.load(path="something/nonexistent.yml")
+    assert len(a_cassette) == 0
+
+
+def test_cassette_load_invalid_unicode(tmpdir):
+    a_file = tmpdir.join("invalid_unicode.yml")
+    with open(a_file, "wb") as fd:
+        fd.write(b"\xda")
+    with pytest.raises(UnicodeDecodeError):
+        Cassette.load(path=a_file)
```

Comment:
With `load_cassette` fixed to handle `UnicodeDecodeError` I believe this test should then be:

```python
assert Cassette.load(path=a_file) is None
```

(with no more `pytest.raises`)

Reply:
A Cassette is still returned, but with zero records, same as the case where the file doesn't exist. I've modified it to be

```python
    a_cassette = Cassette.load(path=str(a_file))
    assert len(a_cassette) == 0
```

to keep in line with the asserts from `test_cassette_load` and `test_cassette_load_nonexistent`